opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -102,18 +137,22 @@ createReplicaSet() {
 
 #install flow
 installDeps
+scanForNewDisks
 generateCertificate
 configureMongoDB
 
-# only for primary mongodb instance
-if [ $VM_INDEX -eq 0 ]
+# only configure replicasets when clusterSize > 1
+if [ $CLUSTER_SIZE -gt 1 ]
```

Comment:
better being safe than sorry `"$CLUSTER_SIZE"

Reply:
Fixed in https://github.com/opencb/opencga/pull/1109/commits/8f4f7ec0150c82a257beaee174ec332c214f4db8